ddl/ddl_api.go

```diff
@@ -351,6 +372,14 @@ func columnDefToCol(ctx context.Context, offset int, colDef *ast.ColumnDef) (*ta
 	return col, constraints, nil
 }
 
+func columnNameToCol(offset int, colName *ast.ColumnName) (*table.Column, error) {
```

Comment:
This function is too simple to be a function. It is used only in one place and it never return error.